tracer/src/Datadog.Tracer.Native/dd_profiler_constants.h

```diff
@@ -64,9 +64,10 @@ const shared::WSTRING include_assemblies[]{
 };
 
 // Note that this list should be kept in sync with the values in shared/src/Datadog.Trace.ClrProfiler.Native/util.h
-// Note that you also consider adding to the SSI tracer/build/artifacts/requirements.json file
+// Note that you should also consider adding to the SSI tracer/build/artifacts/requirements.json file
 const shared::WSTRING default_exclude_assemblies[]{
```

Comment:
Not now, but we should probably share this `default_exclude_assemblies` list between the two libraries?

Reply:
tbh, we probably don't _really_ need the list here at all anymore as long as we block running _without_ the native loader (which we haven't supported for a long time anyway)

Reply:
> as long as we block running without the native loader

We won't do this in AWS Lambda anymore once the next version of the .NET layer is published.
https://github.com/DataDog/dd-trace-dotnet-aws-lambda-layer/pull/10

Reply:
Oh, looks like that already happened 2 days ago: https://github.com/DataDog/dd-trace-dotnet-aws-lambda-layer/releases/tag/v20

Reply:
Btw, the PR above didn't resolve the issue because the contents of the artifacts changed, so I need to update that again.